### PR TITLE
Release-0.2.0

### PR DESCRIPTION
**Pull request by makaveli10**
- update version in the source code
- release vidsz 0.2.0

This release implements batching. Read frames in a batch with a pre - defined batch size.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,48 +12,76 @@ Backends
...
 from vidsz.opencv import Reader
 
 # open reader
 reader = Reader("static/countdown.mp4")
 
-# print info: width, height, fps etc.
-print(reader)
+# read frame with for loop
+for frame in reader:
+    # use frame however you like
```
Comment (shivamswarnkar) [inline README.md:34]:
```suggestion
    # use ndarry-frame however you like
```

File: README.md [hunk 2]
```diff
@@ -78,14 +106,35 @@ print(writer)
...
+
 
-# write list of frames
-# or directly write everything from reader object
-writer.write_all(reader)
+## Write with ```for-loop```
+```python
+# read frame with for loop
+for frame in reader:
+    # write the frame
```
Comment (shivamswarnkar) [inline README.md:116]:
```suggestion
    # write the ndarry-frame
```

File: README.md [hunk 3]
```diff
@@ -78,14 +106,35 @@ print(writer)
...
+```
+
+
+## Write a ```batch```
+```python
+# read batches and write
+with Reader("dummy.mp4", batch_size=8, dynamic_batch=True) as reader:
+    batch_frames = reader.read()
+    # write list of frames
+    # or directly write everything from reader object
```
Comment (shivamswarnkar) [inline README.md:127]:
```suggestion
```

Comment (shivamswarnkar) [inline README.md:127]:
```suggestion
    # write list or ndarray of frames
```

File: README.md (hunk 1 above)
Comment (shivamswarnkar) [reply README.md:34]: Picked it up from the comment on line 54. I think it can be useful information, what do you think?

**Review by shivamswarnkar — COMMENTED** [review]
Made some minor suggestions, but other than that LGTM.

**Comment by aditya-dl** [comment]
@makaveli10 was the PR updated with changes?

**Comment by makaveli10** [comment]
> @makaveli10 was the PR updated with changes?

Yes.

**Review by shivamswarnkar — APPROVED** [review]

**Comment by makaveli10** [comment]
@aditya-dl the changes that shivam mentioned are updated.

**Pull request merged**